wadas/ui/mainwindow.py

```diff
@@ -287,6 +294,24 @@ def update_toolbar_status(self):
             self.isWindowModified() and bool(self.configuration_file_name)
         )
 
+    def ftp_camera_exists(self):
+        """Method that checks if at least one FTP camera exists in camera list."""
+        for camera in cameras:
+            if camera.type == Camera.CameraTypes.FTP_CAMERA:
+                return True
+        return False
+
+    def email_notifier_exists(self):
+        """Method that checks if email notifier is configured."""
+
+        for notifier in Notifier.notifiers:
+            if (
+                Notifier.notifiers[notifier]
+                and Notifier.notifiers[notifier].type == Notifier.NotifierTypes.EMAIL
+            ):
+                return True
+        return False
```

Comment:
return any(Notifier.notifiers[notifier] and Notifier.notifiers[notifier].type == Notifier.NotifierTypes.EMAIL
                  for notifier in Notifier.notifiers)